code-quality/debugging.md

```diff
@@ -84,13 +84,13 @@ actions:
 
 This will disable the checks for just the current user. The `.trunk/user.yaml` file is specifically gitignored but will be loaded locally if present.
 
-## What should I do if a linter process seems to take longer than expected during a Trunk check?
+## What Should I do if a Linter Process Seems to Take Longer than Expected During a Trunk Check?
```

Comment:
We should ignore this vale error.

Reply:
You mean we should ignore the Title Case rule?

Reply:
I meant trunk-ignore but whatever works